### DISCUSSION TITLE
Tailwind Styling Not Working

### DISCUSSION THREAD

**Discussion by JohnL0919** [tailwindlabs/tailwindcss#15263] [Help]
[https://github.com/JohnL0919/phoenix](url) Hi everyone, a complete noob with tailwind here, I had issues trying to get the styling to work and inlcuding the CDN worked but I am still having trouble getting the theme in my tailwind.config.js to work in my component classes. 

![image](https://github.com/user-attachments/assets/afb6b69b-b601-49a6-aba7-3063a253aa23)
The above is my tailwind.config.js with the styling 

![image](https://github.com/user-attachments/assets/463c9af9-d7b1-40c2-af3c-a782119b762c)
I then include it in the index.css file 

![image](https://github.com/user-attachments/assets/10cfb288-a589-48f5-979e-4b7bba75253f)
But when i use it in the register.jsx it does not get applied properly. 

I have tried many different methods to fix my tailwind config but it still does not work.

**Comment by gaziraihan1** [comment]
Check your tailwindcss is running or not?
Check have you export all type of file in tailwind.config.js file in content:[]?
Check If your img-box class has conflicting styles in another layer (like base or components), this could override your utility styles.
Check by trying import the index.css file in react file.
Try to create a new testComponent and check only className="img-box".
Try to run the tailwind server again, maybe it could work.

**Reply by JohnL0919** [reply]
Hi, thank you for responding. How can I check if tailwindcss is running or not?
